app/src/main/java/com/zeapo/pwdstore/utils/PasswordRepository.kt

```diff
@@ -80,7 +80,9 @@ open class PasswordRepository protected constructor() {
         fun isGitRepo(): Boolean {
             if (repository != null) {
                 // Check if remote exists
-                return repository!!.config.getSubsections("remote").isNotEmpty()
+                return repository!!.config.getSubsections("remote").isNotEmpty() &&
+                        repository!!.objectDatabase.exists() &&
+                        repository!!.allRefs.isNotEmpty()
```

Comment:
Aha!! :+1:

Reply:
Would it make sense to split this into the parts "isGitRepo" (i.e., we can commit files) and "hasRemote" (i.e., we can sync/push/pull)? I'm not sure how we are currently deciding whether we are able to commit newly created password files or not, but this could maybe be cleaned up in one swoop.

Reply:
Not at the moment, we forcibly insert the `origin` remote in most code paths.